src/lib/layouts/workflow-header.svelte

```diff
@@ -145,6 +144,24 @@
       {/if}
     </AccordionLight>
   {/if}
+  {#if currentDetails}
+    <AccordionLight let:open>
+      <div
+        slot="title"
+        class="flex w-full items-center gap-2 rounded p-2 text-xl"
```

Comment:
Looks like this isn't doing anything style wise

```suggestion
        class="flex w-full items-center gap-2 p-2 text-xl"
```